lib/rules/no-insecure-url.js

```diff
@@ -18,11 +18,24 @@ const DEFAULT_EXCEPTIONS = [     // TODO: add more typical false positives such
     /^http:(\/\/|\\u002f\\u002f)schemas\.microsoft\.com(\/\/|\\u002f\\u002f)?.*/i,
     /^http:(\/\/|\\u002f\\u002f)schemas\.openxmlformats\.org(\/\/|\\u002f\\u002f)?.*/i,
     /^http:(\/|\\u002f){2}localhost(:|\/|\\u002f)*/i,
-    /^http:(\/|\\u002f){2}localhost(:|\/|\\u002f)*/i,
     /^http:(\/\/)www\.w3\.org\/1999\/xhtml/i,
     /^http:(\/\/)www\.w3\.org\/2000\/svg/i
 ];
 
+function shouldFix(context, node) {
+  let sourceCode = context.getSourceCode();
+  const text = node.parent
+    ? sourceCode.getText(node.parent)
+    : sourceCode.getText(node);
+  const unfixablePattern = ["insecure", "Insecure", "xmlns"];

```

Comment:
Do we still need xmlns if we have the namespaces added to `DEFAULT_EXCEPTIONS`?

Reply:
We don't if we exempt [xmlns=http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul](https://searchfox.org/mozilla-central/search?q=http%3A%2F%2Fwww.mozilla.org%2Fkeymaster%2Fgatekeeper%2Fthere.is.only.xul&path=test ) (I mean just the url).
I will adjust it

Reply:
Yeah, just add the XUL namespace to the other namespaces